BrickSlider/README.md

```diff
@@ -0,0 +1,37 @@
+# BrickSlider
+
+Brick slider is a web component developed by PROGRAP for sliders.
+
++ touchmove events
++ next and previous button
++ autoscroll
++ optional carousel
+
+## Installation
+
+`$ npm install @prograp/brick-slider`
+
+
+## Usage
+#### Element:
+```
+<brick-banner-slider arrow-controls no-carousel threshold="5" initial-offset="0">
+    <!--Create your elements for navigation  -->
+    <button slot="previous">
+    </button>
+    <button slot="next">
+    </button>
+
+    <!--Put your slides as siblings here-->
+    <img class="picA" />
+    <img class="picB" />
+    <a href="" class="interactive-slide">
+      <img class="picC" />
+    </a>
+</brick-banner-slider>
+```
+#### Attributes:
++ `arrow-controls` **Enables arrow controls**
```

Comment:
We currently do not really support this, so better remove it until we actually make it work or completely remove it.